src/pybind/mgr/rook/rook_cluster.py

```diff
@@ -725,7 +744,7 @@ def _execute_blight_job(self, ident_fault: str, on: bool, loc: orchestrator.Devi
                 deleted = not api_response.items
                 if retries > 5:
                     sleep(0.1)
-                ++retries
+                retries = retries + 1
```

Comment:
```suggestion
                retries += 1
```

also, might want to move this change into a separate commit?

Reply:
done